diagnostic.md

```diff
@@ -34,19 +34,19 @@ Open up this file in Atom, and write your answers below (where indicated).
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+Relative path. An absolute would be like http://blah_blah.com
```

Comment:
Absolute. Reason being when a file starts with `/Users` it starts are the root meaning it's absolute.